### PR TITLE
 Add format and compression to COPY command

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by graingert** [comment]
@cpcloud can you add a unittest for this change please?

**Comment by cpcloud** [comment]
@graingert hey thanks for pinging me, just read the discussions on the status of this dialect. interested in helping maintain it

**Comment by cpcloud** [comment]
adding requested test

File: redshift_sqlalchemy/dialect.py [hunk 1]
```diff
@@ -276,12 +279,13 @@ def visit_copy_command(element, compiler, **kw):
     return """
            COPY %(schema_name)s.%(table_name)s FROM '%(data_location)s'
            CREDENTIALS 'aws_access_key_id=%(access_key)s;aws_secret_access_key=%(secret_key)s%(session_token)s'
-           CSV
```
Comment (graingert) [inline redshift_sqlalchemy/dialect.py]: I'm pretty sure this is possible to do with sqlalchemy.text

Comment (cpcloud) [inline redshift_sqlalchemy/dialect.py]: cool i'm going to submit a new pr that has current master merged into, let's discuss there

Comment (graingert) [inline redshift_sqlalchemy/dialect.py]: you can just reset your remote branch (pr/27) to the commit you want to push

**Comment by cpcloud** [comment]
I think this can be closed now

**Pull request closed without merging**